src/components/NewWorkspaceModal.js

```diff
@@ -106,8 +111,23 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
     Ajax(signal).Groups.list().then(setAllGroups)
   ]))
 
+  const loadBucketLocation = _.flow(
+    withErrorReporting('Error loading data'),
+    Utils.withBusyState(setLoading)
+  )(async () => {
+    const [locationResponse] = await Promise.all([
+      Ajax(signal).Workspaces.workspace(namespace, cloneWorkspace.workspace.name).checkBucketLocation(cloneWorkspace.workspace.googleProject, cloneWorkspace.workspace.bucketName)
+    ])
```

Comment:
is `Promise.all` necessary since there seems to be a single `Promise` is being generated?

```suggestion
    const locationResponse = await Ajax(signal).Workspaces.workspace(namespace, cloneWorkspace.workspace.name).checkBucketLocation(cloneWorkspace.workspace.googleProject, cloneWorkspace.workspace.bucketName)
```

Reply:
Done.